### DISCUSSION TITLE
How to set a no color in daisyUI theme?

### DISCUSSION THREAD

**Discussion by myhendry** [saadeghi/daisyui#1832] [Q&A]
![Screenshot 2023-05-22 at 3 14 18 PM](https://github.com/saadeghi/daisyui/assets/20642843/f4b17444-82ff-4394-9307-0aefdbb13387)

Hi. I am using daisyUI theme and a background image for my website.

I like to set no color in my theme so it can show my background image rather than a full color navbar. 

Is it possible? If so, how do I go about doing so? Thanks

**Comment by saadeghi** [comment]
I'm not sure how you're using the navbar. Providing a code example would help.  
If you have navbar inside the same div that has the background image, I think you have a `bg-*` class for the `navbar`. Removing that class will make navbar transparent and you can see the background image.  
If the background image is for an element that is coming after the navbar, you should either use the background image on the parent element of the navbar instead or make the navbar `fixed` so the navar goes on top of the background image (not after it)

Let me know if you have any questions.
